draft-ietf-dnsop-svcb-https.md

```diff
@@ -736,10 +736,8 @@ ALPNs are identified by their registered "Identification Sequence"
 
     alpn-id = 1*255OCTET
 
-"alpn" is a multi-valued SvcParamKey.  To construct its value-list, apply the
-value-list decoding algorithm ({{value-list}}) to the SvcParamValue.
-Each decoded value in the "alpn" value-list
-SHALL be an `alpn-id`.  The value-list MUST NOT be empty.
```

Comment:
Do we still need the "value-list MUST NOT be empty" constraint?

Reply:
It's still there: "one or more".